app/views/layouts/header.html.erb

```diff
@@ -0,0 +1,7 @@
+<nav class="navbar navbar-light bg-light">
+  <!--- giving profiles needs to be two lines - do we need a logo or should we do this in text? -->
+  <a class="navbar-brand" href="#">Giving Profiles</a>
+</nav>
+
+<!--enables ion icons -->
+<script src="https://unpkg.com/ionicons@4.4.4/dist/ionicons.js"></script>
```

Comment:
I may be wrong, but I thought we agreed to use FontAwesome for icons.

Reply:
Oh yeah maybe. I might have mixed those up.